Kernel/Debug.h.in

```diff
@@ -127,8 +127,8 @@
 #cmakedefine01 ICMP_DEBUG
 #endif
 
-#ifndef INTEL_GRAPHICS_DEBUG
-#cmakedefine01 INTEL_GRAPHICS_DEBUG
+#ifndef INTEL_GPU_DEBUG
+#cmakedefine01 INTEL_GPU_DEBUG
```

Comment:
Same here, probably

Reply:
This is intentional, because future generations are not merely about graphics (2D) but it's an entire processor unit.